backend/src/login/index.js

```diff
@@ -100,12 +103,11 @@ function loginSuccessful(req, res, next, ldapUser) {
       aud: config.application.host,
       uid: users.get(ldapUser).id
     }
-    const hasVault = users
-      .get(ldapUser)
-      .getGroupNames()
-      .some(g => g.endsWith('vault'))
 
-    if (hasVault) {
+    const userGroups = users.get(ldapUser).getGroupNames()
+
+    // If user is in the vault group, get vault token
+    if (userGroups.some(g => g.endsWith('vault'))) {
```

Comment:
It's a bit of a pitty that you have to repeat the `some(... endsWith(...))` thingy twice. How about something like
```
const isMember = group => users.get(...).some(g => g.endsWith(group))
// ...
isMember('vault')
```

Reply:
yes sir